### PR TITLE
[ADD] l10n_ca_account_profile

### PR DESCRIPTION
**Pull request by max3903**
Specified at https://blueprints.launchpad.net/openerp-canada/+spec/l10n-ca-account-profile

### CODE REVIEW

**Comment by bwrsandman** [comment]
@max3903 which repositories are dependencies of this module?

They have to be added to https://github.com/OCA/l10n-canada/blob/7.0/.travis.yml#L15

**Comment by max3903** [comment]
bzr lp:~savoirfairelinux-openerp/openerp-usa/npg_bank_account_reconciliation-add_unclear_entries_summation ../../../bzr/openerp-usa last:1
git https://github.com/OCA/account-financial-reporting.git ../../../git/account-financial-reporting 7.0
git https://github.com/OCA/account-financial-tools.git ../../../git/account-financial-tools 7.0
git https://github.com/OCA/bank-statement-reconcile.git ../../../git/bank-statement-reconcile 7.0
git https://github.com/OCA/webkit-tools.git ../../../git/webkit-tools 7.0

**Comment by max3903** [comment]
A repository is missing in Travis:
lp:~noviat/openobject-addons/extra-7.0

**Comment by max3903** [comment]
:+1:

File: hr_expense_line_supplier/hr_expense_line_supplier.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
```
Comment (bwrsandman) [inline hr_expense_line_supplier/hr_expense_line_supplier.py]: File should be called `hr_expense_line_supplier/hr_expense_line.py`

File: account_tax_expense_include/account_tax_expense_include.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# -*- encoding: utf-8 -*-
```
Comment (bwrsandman) [inline account_tax_expense_include/account_tax_expense_include.py]: File should be called `account_tax_expense_include/account_tax.py` (after the class/model)

**Comment by bwrsandman** [comment]
:+1:

**Comment by foutoucour** [comment]
:+1:

**Pull request merged**